docs/conf.py

```diff
@@ -393,10 +393,10 @@ def resig(app, what, name, obj, options, signature, return_annotation):
     in configurator """
     docobj = getattr(obj, '__docobj__', None)
     if docobj is not None:
-        argspec = inspect.getargspec(docobj)
-        if argspec[0] and argspec[0][0] in ('cls', 'self'):
-            del argspec[0][0]
-        signature = inspect.formatargspec(*argspec)
+        params = list(inspect.signature(docobj).parameters.values())
+        if params and params[0].name in ('cls', 'self'):
+            del params[0]
+        signature = str(inspect.Signature(params))
```

Comment:
Is this actually used or displayed in the docs? I've always mentally "bleeped" over it when updating docs. So I tried to make sense of what this actually does, but I just don't grok it. Next I tried looking at any decorated method, such as https://docs.pylonsproject.org/projects/pyramid/en/latest/api/config.html#pyramid.config.Configurator.add_route but I don't see anything related to `action_method`.